policy_test.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/TykTechnologies/tyk/config"
+)
+
+func TestLoadPoliciesFromDashboardReLogin(t *testing.T) {
+	// Mock Dashboard
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(403)
+	}))
+	defer ts.Close()
+
+	oldUseDBAppConfigs := config.Global.UseDBAppConfigs
+	config.Global.UseDBAppConfigs = false
+
+	defer func() {
+		config.Global.UseDBAppConfigs = oldUseDBAppConfigs
+	}()
+
+	connStr := ts.URL
+	secret := "supersecret"
```

Comment:
I would just use an empty string directly in the call, given how it's not important.